exercise_0/fileMode/FileMode.cpp

```diff
@@ -0,0 +1,22 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+
+#include "FileMode.h"
+#include <iostream>
+#include <fstream>
+
+int FileMode::run(const string &path) {
+    ifstream  file;
```

Comment:
Please note, that you might have used c-tor (2 or any other following, which would suit your needs) - https://en.cppreference.com/w/cpp/io/basic_ifstream/basic_ifstream effectively, limiting those two lines into one.